x/metoken/keeper/intest/interest_test.go

```diff
@@ -0,0 +1,133 @@
+package keeper_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	sdkmath "cosmossdk.io/math"
```

Comment:
We should always differentiate between sdk packages and the built-in ones to avoid confusion. `sdkerrors` for `github.com/cosmos/cosmos-sdk/types/errors`, `sdkmath` for `cosmossdk.io/math` and so on.